dbapi/src/main/java/com/griddynamics/jagger/dbapi/DatabaseServiceImpl.java

```diff
@@ -1403,72 +1034,62 @@ public DetailsNode call() {
-                parameters.put("CPU number of cores",String.valueOf(nodeInfoEntity.getCpuTotalCores()));
-                parameters.put("CPU number of sockets",String.valueOf(nodeInfoEntity.getCpuTotalSockets()));
+                parameters.put("CPU model", nodeInfoEntity.getCpuModel());
+                parameters.put("CPU frequency, MHz", String.valueOf(nodeInfoEntity.getCpuMHz()));
+                parameters.put("CPU number of cores", String.valueOf(nodeInfoEntity.getCpuTotalCores()));
+                parameters.put("CPU number of sockets", String.valueOf(nodeInfoEntity.getCpuTotalSockets()));
                 parameters.put("Jagger Java version", nodeInfoEntity.getJaggerJavaVersion());
                 parameters.put("OS name", nodeInfoEntity.getOsName());
                 parameters.put("OS version", nodeInfoEntity.getOsVersion());
-                parameters.put("System RAM, MB",String.valueOf(nodeInfoEntity.getSystemRAM()));
+                parameters.put("System RAM, MB", String.valueOf(nodeInfoEntity.getSystemRAM()));
 
                 List<NodePropertyEntity> nodePropertyEntityList = nodeInfoEntity.getProperties();
                 for (NodePropertyEntity nodePropertyEntity : nodePropertyEntityList) {
-                    parameters.put("Property '" + nodePropertyEntity.getName() + "'",nodePropertyEntity.getValue());
+                    parameters.put("Property '" + nodePropertyEntity.getName() + "'", nodePropertyEntity.getValue());
                 }
-                NodeInfoDto nodeInfoDto = new NodeInfoDto(nodeInfoEntity.getNodeId(),parameters);
+                NodeInfoDto nodeInfoDto = new NodeInfoDto(nodeInfoEntity.getNodeId(), parameters);
 
                 String sessionId = nodeInfoEntity.getSessionId();
                 if (sessions.containsKey(sessionId)) {
                     sessions.get(sessionId).add(nodeInfoDto);
-                }
-                else {
-                    List <NodeInfoDto> node = new ArrayList<NodeInfoDto>();
+                } else {
+                    List<NodeInfoDto> node = new ArrayList<>();
                     node.add(nodeInfoDto);
                     sessions.put(sessionId, node);
                 }
             }
 
-            for (Map.Entry<String,List<NodeInfoDto>> session : sessions.entrySet()) {
-                nodeInfoPerSessionDtoList.add(new NodeInfoPerSessionDto(session.getKey(),session.getValue()));
-            }
+            nodeInfoPerSessionDtoList.addAll(sessions.entrySet().stream()
+                    .map(session -> new NodeInfoPerSessionDto(session.getKey(), session.getValue())).collect(Collectors.toList()));
 
             log.info("For session ids " + sessionIds + " were found node info values in " + (System.currentTimeMillis() - time) + " ms");
-        }
-        catch (NoResultException ex) {
+        } catch (PersistenceException ex) {
             log.info("No node info data was found for session id " + sessionIds, ex);
```

Comment:
Because NoResultException extends PersistenceException and action on both of them is the same.